test/pls.jl

```diff
@@ -118,16 +118,11 @@ end
     print(io, fm1)
     @test startswith(String(take!(io)), "Linear mixed model fit by REML")
 
-    fm1.optsum.maxfeval = 5
-    fm1.optsum.feval = -1
-    @test_logs (:warn, "NLopt optimization failure: MAXEVAL_REACHED") fit!(fm1)
-    fm1.optsum.maxfeval = -1
-
     vc = fm1.vcov
     @test isa(vc, Matrix{Float64})
-    @test only(vc) ≈ 409.79495436473167 rtol=1.e-6
+    @test only(vc) ≈ 375.7167775 rtol=1.e-6
```

Comment:
This changed because we're using the fully fitted model and not prematurely terminated one.